### ISSUE TITLE
Third-party license notices missing OpenMPT

### ISSUE THREAD

**Issue by sagamusix** [neumatho/NostalgicPlayer#88]
I generally don't care if people copy over a couple lines of code and use my module loaders as a reference / documentation source for their own loaders and then don't do any attribution, but the recently-added MO3 decoder is clearly a 1:1 copy of pretty much the entire MO3 loader from OpenMPT. It would be nice if you could add a reference here: https://github.com/neumatho/NostalgicPlayer?tab=readme-ov-file#license
Thanks.

**Comment by neumatho** [comment]
Yes, you're right that parts of my MO3 converter are copied from OpenMPT, specially the decruncher and some parts of the loaders. The savers are written by me. I have credited you, which can be shown in the player as you can see here (around line 150):

https://github.com/neumatho/NostalgicPlayer/blob/main/Source/Agents/ModuleConverters/Mo3Converter/Resources.resx

However, I don't have any problems adding a license, so I have done that too.

**Comment by sagamusix** [comment]
Thanks!
